release-content/0.16/release-notes/_release-notes.toml

```diff
@@ -71,6 +71,13 @@ contributors = []
 prs = [16427]
 file_name = "16427_Use_multidraw_for_opaque_meshes_when_GPU_culling_is_in_use.md"
 
+[[release_notes]]
+title = "Occlusion culling"
+authors = ["@JMS55", "@pcwalton"]
+contributors = []
+prs = [12898, 17413, 17934, 17951]
```

Comment:
12898 is not new this cycle, but is IMO a vital part of the story here. Ordinarily I wouldn't cite previous work that laid foundations, but this is an adaption of an existing in-Bevy design to a new domain.

Both of those are hard, useful and worth celebrating, and occlusion culling for meshlets was not previously explicitly called out as an accomplishment.

Reply:
So actually occlusion culling was first added in https://github.com/bevyengine/bevy/pull/12755 (along with the rest of the initial virtual geometry renderer, it was a very very large PR). And then there were a couple of bugfix PRs later on, and then 12898 which switched the method we used for keeping track of what was visible last frame or not.